Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/MotorMachineInterface.mo

```diff
@@ -0,0 +1,106 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors.BaseClasses;
+model MotorMachineInterface
+  "Calculates the electromagnetic torque based on voltage and frequency"
+
+  parameter Integer pole=2 "Number of pole pairs";
+  parameter Integer n=3 "Number of phases";
+  parameter Modelica.Units.SI.Resistance R_s=0.013 "Electric resistance of stator";
+  parameter Modelica.Units.SI.Resistance R_r=0.009 "Electric resistance of rotor";
+  parameter Modelica.Units.SI.Reactance X_s=0.14 "Complex component of the impedance of stator";
+  parameter Modelica.Units.SI.Reactance X_r=0.12 "Complex component of the impedance of rotor";
+  parameter Modelica.Units.SI.Reactance X_m=2.4 "Complex component of the magnetizing reactance";
+
+  Real s(min=0,max=1) "Motor slip";
+  Modelica.Units.SI.AngularVelocity omega_s "Synchronous angular velocity";
+
+  Modelica.Blocks.Interfaces.RealInput V_rms(unit="V") "Prescribed RMS voltage"
+    annotation (Placement(transformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,40}),
+                         iconTransformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,40})));
+  Modelica.Blocks.Interfaces.RealInput f(final quantity="Frequency", final unit="Hz")
+    "Controllale freuqency to the motor"
```

Comment:
typo `Controllale`.

Reply:
Thank you for catching this, I have corrected the typo.